app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,27 @@
  *= require foundation_and_overrides
 
  */
+
+ /*= require foundation */
+
+
+
+.contain-image {
+    height: 200px;
+    width: 300px;
+    overflow: hidden;
+    object-position: center;
+}
+
+.card {
+    text-align: center;
+    margin: 10px;
+}
+
+.card-title {
+    text-align: center;
+}
+
+.webpage {
+    padding: 15px;
+}
```

Comment:
I left some comments on the previous PR from this group around CSS so you may want to sync up with your group about handling CSS. long story short, we shouldn't ne modifying this file and should create separate files for styling!